presto-native-execution/presto_cpp/main/operators/PartitionAndSerialize.cpp

```diff
@@ -309,6 +404,15 @@ class PartitionAndSerializeOperator : public Operator {
   std::vector<uint32_t> partitions_;
   // Reusable vector for storing serialised row size for each input row.
   std::vector<uint32_t> rowSizes_;
+
+  // For processing null skewed input vector.
+  // Keep reference of serialized data vector, this is used when processing
+  // subsequent batches.
+  Buffer* serializedDataVector;
```

Comment:
naming: member variables should have trailing _

Please, check out coding guidelines and update as needed.